share/rcm.sh.in

```diff
@@ -147,7 +147,13 @@ de_dot() {
   echo "$1" | sed -e "s|$DEST_DIR/||" | sed -e 's/^\.//'
 }
 
-: ${RCRC:=$HOME/.rcrc}
+if [ -z "$RCRC" ]; then
+  if [ -r "$PWD/rcrc" ]; then
+    RCRC=$PWD/rcrc
+  else
+    RCRC=$HOME/.rcrc
```

Comment:
Quotes around these values (`RCRC="$PWD/rcrc"` and `RCRC="$HOME/.rcrc"`), please.